tools/travis-before-install.sh

```diff
@@ -33,5 +36,6 @@ python -V
 popd
 
 pip install --upgrade pip setuptools
-pip install -r test_requirements.txt
+pip install Cython==0.29.14
```

Comment:
We need to keep this in a text file in order for @dependabot to find it I think. Perhaps we need a `build_requires.txt`, which could also include a setuptools version?

Reply:
The `requires` section in `pyproject.toml` lists the build requirements.  I just now getting up to speed on which tools actually use that file.  It would be nice if we could have that list in just one place.  (Python packaging... $%@#@#^$$*@#$!)